Docs/source/running_cpp/parameters.rst

```diff
@@ -1108,6 +1108,32 @@ Diagnostics and output
     Use a negative number to disable data dumping.
     This is ``-1`` (disabled) by default.
 
+    * ``<species_name>.random_fraction`` (`float`) optional
+        If provided ``<species_name>.random_fraction = a``,
+        only `a` fraction of the particle data of this species will be dumped randomly,
+        i.e. if `rand() < a`, this particle will be dumped,
+        where `rand()` denotes a random number generator.
+        The value `a` provided should be between 0 and 1.
+
+    * ``<species_name>.uniform_stride`` (`int`) optional
+        If provided ``<species_name>.uniform_stride = n``,
+        every `n` particle of this species will be dumped,
+        i.e. if `particle ID % n == 0`, this particle will be dumped.
+        The value provided should be an integer greater than or equal to 0.
+
+    * ``<species_name>.plot_filter_function(t,x,y,z,ux,uy,uz)`` (`string`) optional
+        User can provide an expresion that describs the condition to filter
+        a portion of particles that will be dumped.
```

Comment:
It is OK to write lines >100 char in rst file, otherwise reformatting is cumbersome. 
```suggestion
        Users can provide an expression returning a boolean for whether a particle is dumped (the exact test is whether the return value is `> 0.5`).
```